lib/mina_sidekiq/tasks.rb

```diff
@@ -40,9 +40,18 @@
 set :sidekiq_timeout, 11
 
 # ### sidekiq_config
-# Sets the path to the configuration file of sidekiq
+# Sets the path to the configuration file of sidekiq.
 set :sidekiq_config, -> { "#{fetch(:current_path)}/config/sidekiq.yml" }
 
+# ### sidekiq_configs
+# Sets the path to the configurations files of sidekiq when using more than one sidekiq process.
```

Comment:
Please also mention that this conflicts with `sidekiq_config` (mutual exclusive)

Reply:
```
Sets the path to the configurations files of sidekiq when using more than one sidekiq process. This configuration will ignore sidekiq_config.
```

What do you think?